src/coreclr/jit/lower.cpp

```diff
@@ -6759,7 +6759,7 @@ bool Lowering::NodesAreEquivalentLeaves(GenTree* tree1, GenTree* tree2)
 bool Lowering::CheckMultiRegLclVar(GenTreeLclVar* lclNode, const ReturnTypeDesc* retTypeDesc)
 {
     bool canEnregister = false;
-#if FEATURE_MULTIREG_RET
+#if FEATURE_MULTIREG_RET || FEATURE_HW_INTRINSICS
```

Comment:
This does not look right, `FEATURE_HW_INTRINSICS` is a `#define` -- so presumably this should be `|| defined(FEATURE_HW_INTRINSICS)`.